crates/bevy_crevice/src/std430/traits.rs

```diff
@@ -23,9 +23,11 @@ pub unsafe trait Std430: Copy + Zeroable + Pod {
     /// See <https://www.khronos.org/registry/OpenGL/specs/gl/glspec45.core.pdf#page=159>
     /// (rule 4 and 9)
     const PAD_AT_END: bool = false;
-    /// Padded type (Std430Padded specialization)
+    /// Padded type ([`Std430Padded`] specialization)
     /// The usual implementation is
+    /// ```
     /// type Padded = Std430Padded<Self, {align_offset(size_of::<Self>(), ALIGNMENT)}>;
+    /// ```
```

Comment:
Oh looks like the CI wants to run this as code now. I guess I'll just put it in single backticks then